app/views/groups/_form.html.erb

```diff
@@ -0,0 +1,8 @@
+<%= simple_form_for [@draw, @group] do |f| %>
```

Comment:
Ohhh I didn't realize that it was a nested route when I first looked here, so that's why I was confused. Now it makes sense.